<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plaview2pui</class>
 <widget class="QWidget" name="plaview2pui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(245, 246, 250);
</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>49</y>
     <width>451</width>
     <height>511</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(196, 225, 255);
color: rgb(12, 36, 97);</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>449</width>
      <height>509</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="namen">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>70</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(196, 225, 255);
color: rgb(12, 36, 97);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="typen">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>140</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(196, 225, 255);
color: rgb(12, 36, 97);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="timen">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>210</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(196, 225, 255);
color: rgb(12, 36, 97);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="locationn">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>280</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(196, 225, 255);
color: rgb(12, 36, 97);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>530</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(196, 225, 255);
color: rgb(12, 36, 97);</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="tips">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>350</y>
     <width>171</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(196, 225, 255);
color: rgb(12, 36, 97);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>70</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(12, 36, 97);</string>
   </property>
   <property name="text">
    <string>Event Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>140</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(12, 36, 97);</string>
   </property>
   <property name="text">
    <string>Event Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(12, 36, 97);</string>
   </property>
   <property name="text">
    <string>Event Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>280</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(12, 36, 97);</string>
   </property>
   <property name="text">
    <string>Event Location</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>355</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(12, 36, 97);</string>
   </property>
   <property name="text">
    <string>Instructions</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>20</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(196, 225, 255);
color: rgb(12, 36, 97);</string>
   </property>
   <item>
    <property name="text">
     <string>Normal Mode</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Simulation</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(12, 36, 97);</string>
   </property>
   <property name="text">
    <string>Events Feed</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
